grc/core/generator/flow_graph.py.mako

```diff
@@ -11,18 +11,24 @@ python_version = version_info.major
 % endif
 # -*- coding: utf-8 -*-
 <%def name="indent(code)">${ '\n        '.join(str(code).splitlines()) }</%def>
-"""
-GNU Radio Python Flow Graph
-
-Title: ${title}
+#
+# SPDX-License-Identifier: GPL-3.0
+#
+##################################################
+# GNU Radio Python Flow Graph
+# Title: ${title}
 % if flow_graph.get_option('author'):
-Author: ${flow_graph.get_option('author')}
+# Author: ${flow_graph.get_option('author')}
+% endif
+% if flow_graph.get_option('copyright'):
+# Copyright: ${flow_graph.get_option('copyright')}
 % endif
 % if flow_graph.get_option('description'):
-Description: ${flow_graph.get_option('description')}
+# Description: ${flow_graph.get_option('description')}
 % endif
-Generated: ${ generated_time }
-"""
+# Generated: ${generated_time}
```

Comment:
Ooops, this must have slipped my attention in the original commit (pre-forward-to-YAML-porting):

We can not include generation times! That breaks reproducible buildability, since we can generate python flowgraphs at build time, and then the resulting files aren't bitwise identical for the exact same thing compiled on the exact same platform.

Reply:
For flowgraphs generated at run time this feature might be pretty handy. Maybe we want to introduce a flag/option which will be set during build time to skips the time insertion.

Reply:
But this should be handled in another PR.